### PR TITLE
Fixes #12885 - Remove forcing 'no whitespace' from policy name

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

**Comment by shlomizadok** [comment]
Will fail on engine error.... :/

File: app/models/foreman_openscap/policy.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ class Policy < ActiveRecord::Base
     SERVER_CLASS_PARAMETER   = 'server'
     PORT_CLASS_PARAMETER     = 'port'
 
-    validates :name, :presence => true, :uniqueness => true, :format => { :without => /\s/ }
+    validates :name, :presence => true, :uniqueness => true
```
Comment (ohadlevy) [inline app/models/foreman_openscap/policy.rb]: is there a reason why this existed in the first place? any limitation on scap scanner itself?

Comment (shlomizadok) [inline app/models/foreman_openscap/policy.rb]: Not as far as I know. And we use this name only to display it on foreman.
Maybe in the past it was used to set the puppet? We switched to id long time ago though

**Comment by shlomizadok** [comment]
[test]

**Comment by ares** [comment]
the test is failing because of this change, seems like it was on purpose

**Comment by shlomizadok** [comment]
@ares, not sure [test] was unstable. (also tested locally and it worked)

**Comment by ares** [comment]
let's see again but I think the [test] would have to be fixed too, IIRC it was no intermittent failure

**Comment by xprazak2** [comment]
@shlomizadok could you rebase pls? I will set this as WoC for now...

**Comment by shlomizadok** [comment]
@xprazak2 - rebase and fixed the test :)

**Comment by xprazak2** [comment]
Works for me, we can merge if there are no objections.

**Comment by ares** [comment]
:+1:

**Comment by shlomizadok** [comment]
Thanks @ares & @xprazak2  for reviewing and testing

**Pull request merged**